docs/api/indexing.md

```diff
@@ -0,0 +1,27 @@
+# Indexing functions
+
+## geoToH3
+
+```
+H3Index geoToH3(const GeoCoord *g, int res);
+```
+
+Indexes the location at the specified resolution.
+
+Returns 0 on error.
+
+## h3ToGeo
+
+```
+void h3ToGeo(H3Index h3, GeoCoord *g);
+```
+
+Finds the centroid of the index.
```

Comment:
What do we actually do on errors here?

Reply:
It is undefined. Several functions, such as h3ToGeo, h3ToGeoBoundary and kRing don't really have ways to indicate errors.